include/deal.II/fe/fe_base.h

```diff
@@ -36,28 +36,22 @@ namespace FiniteElementDomination
    * An enum that describes the outcome of comparing two elements for mutual
    * domination. If one element dominates another, then the restriction of the
    * space described by the dominated element to a face of the cell is
-   * strictly larger than that of the dominating element. For example, in 2-d
+   * strictly larger than that of the dominating element. For example,
    * Q(2) elements dominate Q(4) elements, because the traces of Q(4) elements
    * are quartic polynomials which is a space strictly larger than the
```

Comment:
this text talks about the trace, so it's about *face* domination. maybe mention this in the comment here, and then extend it to say that the Q2 of course also dominates the Q4 on *cells*.